llama-index-integrations/readers/llama-index-readers-feishu-wiki/README.md

```diff
@@ -0,0 +1,23 @@
+# Feishu Wiki Loader
+
+This loader can traverse all feishu documents under the feishi space.
+
+## Usage
+
+To use this loader, you need to:
+1. apply the permission(`wiki:wiki:readonly`) of the feishu app
+2. add the feishu app as the admin of your feishu space, see [here](https://open.feishu.cn/document/server-docs/docs/wiki-v2/wiki-qa#b5da330b) for more help
+3. finally, pass your feishu space id to this loader
+
+```python
+from llama_index import download_loader
+
+app_id = "cli_slkdjalasdkjasd"
+app_secret = "dskLLdkasdjlasdKK"
```

Comment:
do you want to keep those?

Reply:
I will copy this from another file in `llama-index-readers-feishu-docs`... I will remove this.